### PR TITLE
Task/hchow/2231 ui changes to admin console

### PR DESCRIPTION
**Pull request by Joy603**

### CODE REVIEW

**Comment by hktest** [comment]
This PR includes code cleanup and testing add/remove functionality of the Account Admin Console.

File: selenium_tests/course_info/course_info_add_people_tests.py [hunk 1]
```diff
@@ -27,14 +25,90 @@ def test_search_and_add_person(self):
         # click on course link to view list of people in course
         search_page.select_course(cid=course['cid'])
```
Comment (rascalking) [inline selenium_tests/course_info/course_info_add_people_tests.py]: Should we be asserting that this succeeds, too?

File: selenium_tests/course_info/course_info_base_test_case.py [hunk 2]
```diff
@@ -55,3 +61,37 @@ def search_for_course(self, type=None, school=None, term=None, year=None,
             self._search_page.select_year(year)
         self._search_page.submit_search(search_term)
 
+    def _api_post(self, path=None, body=None):
```
Comment (rascalking) [inline selenium_tests/course_info/course_info_base_test_case.py]: Just a suggestion for next time, these could all be one call that takes the method as an argument.

File: selenium_tests/course_info/course_info_add_people_tests.py (hunk 1 above)
Comment (rascalking) [inline selenium_tests/course_info/course_info_add_people_tests.py]: Ah, the callers are doing this.  Worth moving it in next time, imo.

**Comment by rascalking** [comment]
:+1:

**Comment by hktest** [comment]
Merging to develop.  Tested.

**Pull request merged**